package.json

```diff
@@ -17,7 +17,7 @@
 	},
 	"sideEffects": false,
 	"engines": {
-		"node": ">=18.19"
+		"node": ">=20"
```

Comment:
Not sure whether you'd prefer to target a specific minor/patch release @sindresorhus ?

Reply:
Yeah, otherwise we get complaints later if we use a 20.1 feature, etc.

Reply:
Maybe 20.18?

Reply:
Done. :+1:

Reply:
Was 20.18 selected for any particular reason? `nano-spawn` is a dependency of https://github.com/igorshubovych/markdownlint-cli and because of this change, the latest version fails to run on pre-commit.ci which has node 20.17 (https://results.pre-commit.ci/run/github/891375622/1747578063.uOpKwXApTwOvXEcr7ycNjQ).

Reply:
I don't think so. `20.18.0` does not have any features that we really need, compared to `20.17.0`.

@sindresorhus What are your thoughts on publishing a patch release that decreases the Node version from `20.18.0` to `20.17.0`, so it works on `pre-commit.ci`?

Reply:
Sure

Reply:
@nijel This should be fixed in [`1.0.2`](https://github.com/sindresorhus/nano-spawn/releases/tag/v1.0.2).